src/Page2/triplebulle.tsx

```diff
@@ -1,41 +1,84 @@
 import React from "react";
+import { useState } from "react";
 import './triplebulle.css'
+<style>
+@import url('https://fonts.googleapis.com/css2?family=Krona+One&display=swap');
+</style>
 
 
-const TripleBulle : React.FC = () => {
-    return(
+
+
+const TourneImage : React.FC  = () => {
+    const [positions, setPositions] = useState(['left','center','right']);
+
+
+    const tourneGauche =() => {
```

Comment:
this is duplicated, better define a common function and use it in both locations